### PR TITLE
:sparkles:[#1630] add experimental communicatiekanaal_naam field

### LINKED ISSUE #1630: Als gebruiker van Open Zaak wil ik een communicatiekanaal kunnen opvoeren
### Thema / Theme

Zaken API

### Omschrijving / Description

VNG heeft de [VNG Referentielijsten gearchiveerd](https://github.com/VNG-Realisatie/vng-referentielijsten). De opvolger is de [Referentielijsten API](https://github.com/VNG-Realisatie/referentielijsten-api) echter deze heeft geen communicatiekanaal meer.

Een Communicatiekanaal is nog wel onderdeel van de Zaken API (1.5). Dit zorgt voor problemen.

Als praktische oplossing gaan we "communicatiekanaalNaam" opnemen als extra EXPERIMENTEEL veld in Open Zaak Zaken API.

### Toegevoegde waarde / Added value

_No response_

### Aanvullende opmerkingen / Additional context

_No response_

### PR DESCRIPTION
**Pull request by Coperh**
Fixes #1630 

* Adds experimental communicatiekanaalNaam

### CODE REVIEW

**Comment by Coperh** [comment]
I'm not 100% sure where I was supposed to put this field.

File: src/openzaak/components/zaken/models/zaken.py [hunk 1]
```diff
@@ -411,6 +412,16 @@ class Zaak(ETagMixin, AuditTrailMixin, APIMixin, ZaakIdentificatie):
             "De naam van de registratie waarvan het procesobject deel uit maakt."
         ),
     )
+    communicatiekanaal_naam = models.CharField(
+        _("communicatiekanaal naam"),
+        max_length=250,
+        blank=True,
+        help_text=mark_experimental(
+            _(
+                "De naam van medium waarlangs de aanleiding om een zaak te starten is ontvangen."
```
Comment (bart-maykin) [inline src/openzaak/components/zaken/models/zaken.py:421]:
```suggestion
                "De naam van de medium waarlangs de aanleiding om een zaak te starten is ontvangen."
```

**Review by bart-maykin — APPROVED** [review]

File: src/openzaak/components/zaken/models/zaken.py (hunk 1 above)
Comment (alextreme) [reply src/openzaak/components/zaken/models/zaken.py:421]: het medium

File: src/openzaak/components/zaken/api/serializers/zaken.py [hunk 2]
```diff
@@ -368,6 +368,7 @@ class Meta:
             "processobjectaard",
             "startdatum_bewaartermijn",
             "processobject",
+            "communicatiekanaal_naam",
```
Comment (stevenbal) [inline src/openzaak/components/zaken/api/serializers/zaken.py:371]: Can you move this line to just below `communicatiekanaal`? That way they show up after eachother in the API schema

**Review by stevenbal — CHANGES_REQUESTED** [review]

File: src/openzaak/components/zaken/api/serializers/zaken.py (hunk 2 above)
Comment (Coperh) [reply src/openzaak/components/zaken/api/serializers/zaken.py]: Moved

**Review by stevenbal — APPROVED** [review]

**Pull request merged**